src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -0,0 +1,745 @@
+---
+title: "NerdGraph tutorial: Manage your synthetic monitors"
+tags:
+    - NerdGraph
+    - Synthetic
+metaDescription: Use NerdGraph to programatically manage your synthetic monitors, private locations, and secure credentials.
+---
+
+Our NerdGraph API allows you to automate creating, updating, or deleting your synthetic monitors.
+
+If you've never used NerdGraph, read [Intro to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) for a more thorough tutorial. If you want to use the UI instead, check out [Add and edit monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors/).
```

Comment:
I wouldn't call the intro a 'tutorial'. there's not much there. I might even get rid of that sentence and just link to the intro from the first 'NerdGraph API' mention in first sentence. think it's implied that this is just one doc in that area.